### PR TITLE
chore: update to 1.3.5

### PR DESCRIPTION
**Pull request by maiieul**
# What is it?

- [ ] Feature / enhancement
- [ ] Bug
- [ ] Docs / tests
- [ ] Other

# Why is it needed?

# Checklist:

- [ ] My code follows the [developer guidelines of this project](https://github.com/qwikifiers/qwik-ui/blob/main/CONTRIBUTING.md)
- [ ] I have performed a self-review of my own code
- [ ] I have ran `pnpm changeset` and documented my changes
- [ ] I have add necessary docs (if needed)
- [ ] Added new tests to cover the fix / functionality

### CODE REVIEW

File: packages/kit-headless/src/components/qwik-ui-provider/qwik-ui-provider.tsx [hunk 1]
```diff
@@ -12,7 +12,7 @@ import {
   type Signal,
   Fragment,
 } from '@builder.io/qwik';
-import { type JSXNode } from '@builder.io/qwik/jsx-runtime';
+import { type JSXNode } from '@builder.io/qwik';
```
Comment (thejackshelton) [inline packages/kit-headless/src/components/qwik-ui-provider/qwik-ui-provider.tsx:15]: Let's just remove this file, and anywhere it's being used. It's an old impl

File: packages/kit-headless/src/components/popover/popover.tsx [hunk 2]
```diff
@@ -4,14 +4,7 @@ import { PopoverImpl } from './popover-impl';
...
 
-type PopoverProps = PopoverImplProps &
-  (
-    | {
-        floating?: false | undefined;
-        anchorRef: never;
-      }
-    | ({ floating?: true } & FloatingProps)
-  );
+type PopoverProps = PopoverImplProps & ({ floating?: true } & FloatingProps);
```
Comment (thejackshelton) [inline packages/kit-headless/src/components/popover/popover.tsx:7]: These types will always include FloatingProps, which is not what we want.

What's wrong with the previous types?

File: packages/kit-headless/src/components/qwik-ui-provider/qwik-ui-provider.tsx (hunk 1 above)
Comment (maiieul) [reply packages/kit-headless/src/components/qwik-ui-provider/qwik-ui-provider.tsx:15]: Alright

File: packages/kit-headless/src/components/popover/popover.tsx (hunk 2 above)
Comment (maiieul) [reply packages/kit-headless/src/components/popover/popover.tsx]: It wasn't working anymore :/

**Comment by thejackshelton** [comment]
Looks good to me! Merging.

**Pull request merged**